includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -186,21 +186,11 @@ public static function display_legacy_deprecation_notice( $plugin_file ) {
 	 *
 	 * @since 4.1.0
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in a future release.
```

Comment:
Nit: This should be 10.2.0.
```suggestion
	 * @deprecated 10.2.0 This method will be removed in a future release.
```